open-company/open-company-web
# Polls

Card: https://trello.com/c/EBOzBTjn/2359-polls

Related PRs:
- [Storage](https://github.com/open-company/open-company-storage/pull/243)
- ~~[Infra](https://github.com/belucid/open-company-infrastructure/pull/128)~~

To test:
- add a poll
- vote a poll
- vote with other users to a poll
- add multiple polls to the same post
- make sure tabbing btw choices work when there are multiple polls
- make sure you can always add some text above and below a poll: add it as first thing, after you typed something, after you added an image or video
- make sure you can always add some text above and below a poll: try editing a post previously saved with a poll at the top or bottom
- checkout polls on mobile (read and edit)
